lhotse/augmentation/lowpass.py

```diff
@@ -0,0 +1,37 @@
+from dataclasses import dataclass
+from typing import Literal
+
+import numpy as np
+import scipy.signal
```

Comment:
Since scipy is an optional lhotse dependency, please move this import inside __call__, and add an import guard (from `lhotse.utils`) before it:

```python
if not is_module_available("scipy"):
    raise ImportError("In order to use Lowpass transforms, run 'pip install scipy'")
```